pkg/util/pki/csr.go

```diff
@@ -221,6 +271,34 @@ func SignCertificate(template *x509.Certificate, issuerCert *x509.Certificate, p
 	return pemBytes.Bytes(), cert, err
 }
 
+func SignCSRTemplate(caCerts []*x509.Certificate, caKey crypto.Signer, template *x509.Certificate) (*issuer.IssueResponse, error) {
```

Comment:
Can you add a comment to the new exported methods in this file, as they will be widely used/shared 😄